framework/source/class/qx/core/MEvent.js

```diff
@@ -230,7 +230,7 @@ qx.Mixin.define("qx.core.MEvent",
         if (oldData === undefined) {
           oldData = null;
         }
-        return this.__Registration.fireNonBubblingEvent(
+        return this.__Registration.fireEvent(
```

Comment:
This looks like a major change that could affect lots of existing apps... Why did it change from non-bubbling to bubbling? (or am I misreading it?)

Reply:
fireEvent is using Registration.fireEvent.
fireDataEvent is using Registration.fireNonBubblingEvent.

2 similar functions with different return values is IMHO no good idea.

fireNonBubblingEvent could'nt return a promise. With reading the qooxdoo spec for bubbling to non-bubbling this should'nt break anything.